chat-plugins/poll.js

```diff
@@ -209,7 +209,7 @@ exports.commands = {
 					clearTimeout(room.poll.timeout);
 					room.poll.timeout = null;
 					room.poll.timeoutMins = 0;
-					return this.add("The timeout for the poll was cleared.");
+					return this.add("The timer for the poll was cleared.");
```

Comment:
Hey I like my function names!
yeah this is something I suck at so feel free to change it. I'm neither a
native speaker or good with words.